libraries/radwordsprocessing/editing/find-and-replace.md

```diff
@@ -1,44 +1,67 @@
 ---
-title: Replace Text and Style
-page_title: Replace Text and Style
-description: Learn how to replace text or styling with WordsProcessing
+title: Find and Replace Text and Style
+page_title: Find and Replace Text and Style
+description: Learn how to find and replace text or styling with WordsProcessing
 slug: radwordsprocessing-editing-find-and-replace
-tags: find,replace
+tags: find,search,replace
 published: True
 position: 5
 ---
 
-# Replace Text and Style
+# Find and Replace Text and Style
 
+**RadWordsProcessing** gives you the ability to search for a string in a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}) instance and replace all matches. The library also allows you to replace the styling of the matches alone.
 
-__RadWordsProcessing__ gives you the ability to search for a string in a [RadFlowDocument]({%slug radwordsprocessing-model-radflowdocument%}) instance and replace all matches. The library also allows you to replace the styling of the matches alone.
+Searching and replacing can be achieved with the help of the [RadFlowDocumentEditor]({%slug radwordsprocessing-editing-radflowdocumenteditor%}) class exposed by the library which works with a specific document. 
 
 
-Replacing can be achieved with the help of the [RadFlowDocumentEditor]({%slug radwordsprocessing-editing-radflowdocumenteditor%}) class exposed by the library which works with a specific document. 
+## Find Text
 
+Finding all instances of a string can be achieved with the **FindAll()** method of **RadFlowDocumentEditor**. The editor exposes two methods accepting different parameters.
```

Comment:
RadFlowDocumentEditor exposes the FindAll() method to enable you to find all instances of a string. You can choose between the following overloads: